core/slice.go

```diff
@@ -878,6 +885,9 @@ func (sl *Slice) updatePhCacheFromDom(pendingHeader types.PendingHeader, termini
 			if (localPendingHeader.Header().Root() != types.EmptyRootHash && nodeCtx == common.ZONE_CTX) || nodeCtx == common.REGION_CTX {
 				block := sl.hc.GetBlockOrCandidateByHash(localPendingHeader.Header().ParentHash())
 				if block != nil {
+					// setting the current state will help speed the process of append
+					// after mining this block since the state will already be computed
+					sl.hc.SetCurrentState(block.Header())
```

Comment:
error from `SetCurrentState` is ignored. Want to make sure that's intentional

Reply:
yeah, this is just to make sure we dont have to compute state while appending block